operator/builtin/output/newrelic/newrelic.go

```diff
@@ -180,64 +186,12 @@ func (nro *NewRelicOutput) feedFlusher(ctx context.Context) {
 		case errors.Is(err, context.Canceled):
 			return
 		case err != nil:
-			nro.Errorf("Failed to read chunk", zap.Error(err))
+			nro.flusher.Errorf("Failed to read chunk", zap.Error(err))
```

Comment:
I think we should revert this line, doesn't make sense to use the flusher's logger here.